PushNotifications/c2dm_urban_airship/README.md

```diff
@@ -1,2 +1,68 @@
 #Demo PushNotifications/c2dm_urban_airship
 
+##Step 1:
+
+Urban Airship for Android uses C2DM for push notifications. In order to begin you first need to register an email id with Google’s C2DM framework which can be found here:
+
+http://code.google.com/android/c2dm/signup.html
+
+you will also be required to register the application’s package name.
+
+##Step 2:
+
+Register with Urban Airship on : http://urbanairship.com/ and wait for the confirmation mail from Google.
+
+##Step 3:
+
+Once you have the confirmation mail from Google informing you that your account can now successfully send push notifications you need to create a new application on Urban Airship. Log into Urban Airship and then create a new application. During creation make sure that you have Push Notification Support enabled and you register the same package name you registered with Google’s servers.
+
+##Step 4:
+
+In order to get the C2DM authorisation token you need to execute a curl command
+
+curl https://www.google.com/accounts/ClientLogin -d Email=your_user -d "Passwd=your_password" -d accountType=GOOGLE -d source=Google-cURL-Example -d service=ac2dm
+
+Enter your password and email in the required fields. The response will resemble this
+
+SID=DQAAAPMAAACMY6LUPhXt_co8ESnZa9_bX17MJJZj2AGlDr4gcNIXxq7OJBMPFiT8ry756bE_0rmx
```

Comment:
That was there before
Am 30.06.2015 12:59 schrieb "Xavi Rigau" <notifications@github.com>:

> In PushNotifications/c2dm_urban_airship/README.md
> <https://github.com/novoda/android-demos/pull/9#discussion_r33561294>:
>
> > +
> > +Register with Urban Airship on : http://urbanairship.com/ and wait for the confirmation mail from Google.
> > +
> > +##Step 3:
> > +
> > +Once you have the confirmation mail from Google informing you that your account can now successfully send push notifications you need to create a new application on Urban Airship. Log into Urban Airship and then create a new application. During creation make sure that you have Push Notification Support enabled and you register the same package name you registered with Google’s servers.
> > +
> > +##Step 4:
> > +
> > +In order to get the C2DM authorisation token you need to execute a curl command
> > +
> > +curl https://www.google.com/accounts/ClientLogin -d Email=your_user -d "Passwd=your_password" -d accountType=GOOGLE -d source=Google-cURL-Example -d service=ac2dm
> > +
> > +Enter your password and email in the required fields. The response will resemble this
> > +
> > +SID=DQAAAPMAAACMY6LUPhXt_co8ESnZa9_bX17MJJZj2AGlDr4gcNIXxq7OJBMPFiT8ry756bE_0rmx
>
> I guess this is not a real token?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/android-demos/pull/9/files#r33561294>.
>